doc/changelog.rst

```diff
@@ -6,6 +6,7 @@ Changes in Version 4.13.0 (2025/05/14)
 
 PyMongo 4.13 brings a number of changes including:
```

Comment:
We still need to remove the "This API is currently in beta" warnings in the async api docs.